udata/harvest/backends/dcat.py

```diff
@@ -76,6 +80,9 @@ def inner_harvest(self):
             self.process_one_datasets_page(page_number, page)
             pages.append((page_number, page))
 
+        for org in self.organizations_to_update:
+            org.count_datasets()
```

Comment:
Are we sure that the `org` has `compute_aggregate_metrics=True` here? Shouldn't we set it before calling `org.count_datasets`?

Reply:
It's set to True in [`Organization.__init__`](https://github.com/opendatateam/udata/pull/3391/files#diff-a19a9cbf1d8470b63726d645e31a91d271e758249d0554d0c423ebc201a0a764R191). Not too nice for a one-trick variable, but since this workaround should go away I figured it's ok. The alternative is to use `hasattr` but I'm not a fan.

Reply:
Right :facepalm: